pkg/dist-cache/dist-cache-metrics.go

```diff
@@ -9,14 +9,16 @@ import (
 
 // DistCacheMetrics holds metrics from DistCache, Olric, DMap statistics.
 type DistCacheMetrics struct {
-	EntriesTotal          *prometheus.GaugeVec
-	DeleteHits            *prometheus.GaugeVec
-	DeleteMisses          *prometheus.GaugeVec
-	GetMisses             *prometheus.GaugeVec
-	GetHits               *prometheus.GaugeVec
-	EvictedTotal          *prometheus.GaugeVec
-	PartitionsCount       *prometheus.GaugeVec
-	BackupPartitionsCount *prometheus.GaugeVec
+	EntriesTotal                 *prometheus.GaugeVec
+	DeleteHits                   *prometheus.GaugeVec
```

Comment:
Please change hits and musses to counters

Reply:
Also, need separate result and global cache hit/miss counters

Reply:
As disussed on a call, those are metrics from Olric stats, so they need to stay as is.